test/write_square2D_test.cpp

```diff
@@ -85,7 +85,9 @@ TEST_CASE("writeAscii_test")
 
   SECTION( "ascii" )
   {
-    REQUIRE_NOTHROW( write( filename, mesh, pointData, cellData ) );
+    AsciiWriter writer;
```

Comment:
why is this now required?

Reply:
Sorry, also a leftover of some attempts, we made during the year, which we forgot to reverse, thanks for reminding us, should be fixed now, hopefully in the new branch ;-)